tox.ini

```diff
@@ -338,3 +310,83 @@ commands =
 	- /bin/sh -c "rm -fr ./*private_key.txt"
 	{posargs}
 
+
+[testenv_multi]
```

Comment:
Separate test env for py {3.7, 3.8, 3.9}

Reply:
there are this many compatibility issues between 3.7-3.9 and 3.10?

Reply:
No only pylint, pytest and mypy has compatibility issue

Reply:
We should open an issue to resolve the compatibility issues, if at all possible.